packages/contracts/contracts/PollFactory.sol

```diff
@@ -20,23 +17,13 @@ contract PollFactory is Params, DomainObjs, IPollFactory {
   /// @inheritdoc IPollFactory
   function deploy(
     uint256 _duration,
-    TreeDepths calldata _treeDepths,
-    PubKey calldata _coordinatorPubKey,
-    address _maci,
-    uint256 _emptyBallotRoot
```

Comment:
let's put this back

Reply:
Do you mean `_emptyBallotRoot`?

Reply:
yes not sure why it highlighted everything